src/components/featureHighlight/index.tsx

```diff
@@ -219,8 +252,8 @@ class FeatureHighlight extends BaseComponent {
   }
 
   getContentPosition() {
-    const {highlightFrame, minimumRectSize, innerPadding} = this.props;
-    const {top, height} = this.targetPosition;
+    const {highlightFrame, minimumRectSize = {height: 0}, innerPadding = 0} = this.props;
```

Comment:
`minimumRectSize` is of type `ReactSize`, is setting only the height as default enough?

Reply:
Yes, since it's only used the height in line 262

Reply:
Yes, I know. I just thought it might give you a type error...